.vib/postgresql/goss/goss.yaml

```
@@ -0,0 +1,29 @@
+command:
+  postgresql-connect:
+    exec: PGPASSWORD=ComplicatedPassword1234 psql -U user -d test_database
```

We should avoid hardcoding those values that can be obtained via ENVs:
```suggestion
    exec: PGPASSWORD="$POSTGRES_PASSWORD" psql -U "$POSTGRES_USER" -d "POSTGRES_DB"
```

> Please double-check the ENVs are the correct ones, I may be mistaking them with similar ones.

---

I double-checked and applied the changes. Thanks a lot!